src/utils/dispute-utils.js

```diff
@@ -446,7 +446,8 @@ export function getRoundFees(round, courtConfig) {
   if (round.number === maxRegularAppealRounds)
     return round.jurorsNumber
       .mul(jurorFee)
-      .div(FINAL_ROUND_WEIGHT_PRECISION.mul(finalRoundReduction).div(PCT_BASE))
+      .div(FINAL_ROUND_WEIGHT_PRECISION)
+      .mul(finalRoundReduction.div(PCT_BASE))
```

Comment:
Depending on what we're expecting (and how the contracts are programmed), doing the `.div()` separately may result in more precision.